pom.xml

```diff
@@ -300,6 +312,7 @@ file comparators, endian transformation classes, and much more.
     <commons.release.isDistModule>true</commons.release.isDistModule>
     <commons.releaseManagerName>Gary Gregory</commons.releaseManagerName>    
     <commons.releaseManagerKey>86fdc7e2a11262cb</commons.releaseManagerKey>
+    <jmh.version>1.21</jmh.version>
```

Comment:
Why did you pick a seemingly random old version? Why not 1.27, which is the current version?

Reply:
@garydgregory when I add this into pom(in the former pr), there be only 1.21. this pr has been delayed for too long lol.

Reply:
@garydgregory done.